lib/renderers/server.js

```diff
@@ -9,14 +9,17 @@ import App from 'components/App';
 const serverRender = async () => {
 	
 	const dataURL = `http://${config.host}:${config.port}/data`;
-	const response = await axios.get(dataURL);		
-	const store = new StateApi(response.data);		
-	
-	return {
-			initialMarkup : ReactDOMServer.renderToString(
-		<App store = {store} />
-	), initialData : response.data 
-	};		
+	const response = await axios.get(dataURL);
+	const dateString = new Date().toString();
+	const store = new StateApi(response.data, dateString) ;		

```

Comment:
There should be some way to use this store. This conversation only meant to see how workflow are working

Reply:
Finally PR moved to in Pending Reviews
This conversation only meant to see how workflow are working